app/dagger/SimpleInjector.java

```diff
@@ -0,0 +1,67 @@
+package dagger;
+
+import play.api.inject.BindingKey;
+import play.inject.Injector;
+import scala.reflect.ClassTag;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+/**
+ * A simple injector with additional classes...
+ */
+public class SimpleInjector implements Injector {
```

Comment:
Can't we use the Scala [`SimpleInjector`](https://github.com/playframework/playframework/blob/8d543c6a476a306f556772d8682b867104028090/framework/src/play/src/main/scala/play/api/inject/Injector.scala#L64-L104) instead?